package/dotnet/dotnet-profile

```diff
@@ -0,0 +1 @@
+export DOTNET_SYSTEM_GLOBALIZATION_INVARIANT=1
```

Comment:
This does not seem to get applied. Neither when relogging nor rebooting. Needed to run this manually to create a new test project.

Reply:
It looks like `/etc/profile` is only sourcing `/etc/profile.d/*.sh` files, whoops. I'll fix that.